lxd/storage_volumes_snapshot.go

```diff
@@ -640,6 +643,31 @@ func autoCreateCustomVolumeSnapshotsTask(d *Daemon) (task.Func, task.Schedule) {
 				continue
 			}
 
+			// Get list of cluster nodes
+			nodes, err := d.gateway.NodeStore().Get(d.gateway.Context())
```

Comment:
@monstermunchkin does this only return online nodes?

Reply:
@monstermunchkin it would probably me more efficient to retrive this node list once, rather than on every iteration of the loop.

Reply:
This is fixed now. It was actually listing all nodes.